README.md

```diff
@@ -1,34 +1,58 @@
-## Welcome
+# Welcome
 
-### About this project
-This project enables you to create Cloudinary URLs for your images.
-Using this SDK, you can apply advanced transformations to your images.
+## About this project
 
+This project enables you to create Cloudinary URLs for your images and videos.
+Using this SDK, you can apply advanced transformations to your images and videos.
 
```

Comment:
Might be worth adding that you can use this in conjunction with one of the front end frameworks and link to that repo too.

Reply:
Added that, and some extra links as well to Angular and React.